PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -322,6 +322,12 @@ def miniAOD_customizeCommon(process):
 
 
 def miniAOD_customizeMC(process):
+    #GenJetFlavourInfos
+    process.load("PhysicsTools.JetMCAlgos.HadronAndPartonSelector_cfi")
+
+    process.selectedHadronsAndPartons.particles = "prunedGenParticles"
+    process.load("PhysicsTools.JetMCAlgos.AK4GenJetFlavourInfos_cfi")
```

Comment:
selectedHadronsAndPartons and ak4GenJetFlavourInfos should be added to the task (as done below)